portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,34 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import java.util.ArrayList;
+
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/plain;");
-    response.getWriter().println("Hello Bianca!");
+    // response.setContentType("text/plain;");
+    // response.getWriter().println("Hello Bianca!");
+
+    // Convert comments to JSON
+    ArrayList<String> comments = new ArrayList<String>();
```

Comment:
A couple of things here:

1. Prefer referring to [objects by their interfaces](go/ej3e-64)(if that go link doesn't resolve it's in the PDF i shared previously - item 64) unless there's a reason not to.
```suggestion
    List<String> comments = new ArrayList<String>();
```

2. In general mutable collections can be a source of bugs. I think that [guava's immutable collections](https://github.com/google/guava/wiki/ImmutableCollectionsExplained) will come in handy (as will guava, generally). No action needed, just more food for thought.